lnbits/core/templates/core/wallet.html

```diff
@@ -101,12 +101,25 @@ <h3 class="q-my-none">
                 ></h5>
               </div>
               <div class="col-auto">
-                <q-btn
-                  flat
-                  color="grey"
-                  @click="exportCSV"
-                  :label="$t('export_csv')"
-                ></q-btn>
+                <q-btn-dropdown color="grey" :label="$t('export_csv')">
+                  <q-list>
+                    <q-item clickable v-close-popup @click.prevent="exportCSV">
+                      <q-item-section>
+                        <q-item-label
+                          >{%raw%}{{$t('export_csv')}}{%endraw%}</q-item-label
+                        >
+                      </q-item-section>
+                    </q-item>
+                    <q-item clickable v-close-popup @click="exportCSVFiat">
+                      <q-item-section>
+                        <q-item-label
+                          >{%raw%}{{$t('export_csv')}}
```

Comment:
- maybe an `export_csv_fiat` label should be created
- `"(fiat)"` cannot be considered universal